irods_browser_app/static/js/irods.js

```diff
@@ -34,6 +34,7 @@ $('#btn-signin-irods').on('click',function(){
 $('#btn-select-irods-file').on('click',function(){
     $('#res_type').val($('#resource-type').val());
     $('#file_struct').children().remove();
+    $('.ajax-loader').hide();
```

Comment:
@selimnairb No, this is not a left-over code, but instead it is a minor fix I slated in for iRODS browser I put in which also shows the same spin-wheel when retrieving collection and data objects from iRODS for browsing purposes. I found I did not hide it properly which will have spin-wheel being displayed from start to finish if the user click browse button again. This one line addition will fix that issue.